packages/core/src/domain/tracekit.spec.ts

```diff
@@ -260,7 +256,7 @@ Error: foo
     ;[false, true].forEach((callOnError) => {
       ;[1, 2].forEach((numReports) => {
         let title = 'it should receive arguments from report() when'
-        title += ` callOnError is ${callOnError}`
+        title += ` callOnError is ${callOnError ? 'true' : 'false'}`
```

Comment:
Typescript will complain:
```
Conversion of type 'boolean' to type 'string' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
```
We would need to do `${callOnError as any as string}`
Do you think it's better than `${callOnError ? 'true' : 'false'}` ?

Reply:
We could probably do `${String(callOnError)}`